staff/mariajesus-bellido/challenges/calculator-app/main.js

```diff
@@ -0,0 +1,132 @@
+
+//Declaración de variable 
+//ResetValue = reset calculadora (C)
+//Transformación del dato una vez recogido (String vacía '') -> valor conseguimos que desaparezca el valor
+
+
+var current = ''
+var aux = ''
+var resetValue = '';
+
+// 1 - Selección de todos los botones 
+// 2 - Selección del span (display)
+
+var buttons = document.querySelectorAll('button');
+var display = document.querySelector('span');
+
+//"aux" recogerá el valor para añadirlo a "current"
+// ahora ya vaciaremos / limpiamos el display con '' (empty string)
+
+aux = current
+current = ''
+
+// Botones del 0 al 9  (orden ascendente) 
+buttons [13].onclick = function () { 
+    current = current + '0'
+    display.innerText = current
+}
+
+buttons [8].onclick = function () { 
+    current = current + '1'
+    display.innerText = current
+}
+
+buttons [9].onclick = function () { 
+    current = current + '2'
+    display.innerText = current
+}
+
+buttons [10].onclick = function () { 
+    current = current + '3'
+    display.innerText = current
+}
+
+buttons [4].onclick = function () { 
+    current = current + '4'
+    display.innerText = current
+}
+
+buttons [5].onclick = function () { 
+    current = current + '5'
+    display.innerText = current
+}
+
+buttons [6].onclick = function () { 
+    current = current + '6'
+    display.innerText = current
+}
+
+buttons [0].onclick = function () { 
+    current = current + '7'
+    display.innerText = current 
+}
+
+buttons [1].onclick = function () { 
+    current = current + '8' 
+    display.innerText = current 
+}
+
+buttons [2].onclick = function () { 
+    current = current + '9'

```

Comment:
Point and coma after the numbers in brakets...I think but not sure

Reply:
done, thanks!